monitoring/ingestion/ingestion-producer-alert.yaml

```diff
@@ -58,14 +58,42 @@ groups:
 
   - alert: NoMessagesQueued
     Expr: |
-      ((sum(metadata_journal_size{namespace="${namespace}", job="${job_ingestion_processor}"} -
-        ingestion_read_offset{namespace="${namespace}", job="${job_ingestion_processor}"}+1)) > 1)
+      ((sum(metadata_journal_size{namespace="${namespace}", job="${job_ingestion_producer}"} -
+        ingestion_read_offset{namespace="${namespace}", job="${job_ingestion_producer}"}+1)) > 1)
       and
-      ((sum(rate(ingestion_objects_queued_total{namespace="${namespace}", job="${job_ingestion_processor}"}[5m]))) == 0)
+      ((sum(rate(ingestion_objects_queued_total{namespace="${namespace}", job="${job_ingestion_producer}"}[5m]))) == 0)
     For:   "5m"
     Labels:
       severity: warning
     Annotations:
       zenko_service: backbeat-ingestion-producer
       description: "Ingestion producer has not queued any messages in the last 5 minutes"
       summary: "Ingestion producer has not queued messages"
+
+  - alert: IngestionProducerSourceErrorRate3Percent
+    Expr: |
+      sum(rate(ingestion_source_operation{status="error", namespace="${namespace}", job="${job_ingestion_producer}"}[5m]))
```

Comment:
Should the rate be `number of errors/number of calls` instead of `number of errors/number of successes`?